drivers/soundwire/intel.c

```diff
@@ -835,7 +837,7 @@ static int intel_prepare(struct snd_pcm_substream *substream,
 		return -EIO;
 	}
 
-	return sdw_prepare_stream(dma->stream);
+	return 0;
```

Comment:
you could delete intel_prepare if you do this

Reply:
@RanderWang Yes, thanks.